book/src/drivers.md

```diff
@@ -69,6 +69,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [vSphere](https://github.com/kubernetes-sigs/vsphere-csi-driver) | `vsphere.csi.vmware.com` | v1.0 | A Container Storage Interface (CSI) Driver for VMware vSphere | Persistent | Read/Write Single Pod | Yes | Raw Block
 [YanRongYun](http://www.yanrongyun.com/) | ? | v1.0 | A Container Storage Interface (CSI) Driver for YanRong YRCloudFile Storage  | Persistent | Read/Write Multiple Pods | Yes | 
 [Zadara-CSI](https://github.com/zadarastorage/zadara-csi) | `csi.zadara.com` | v1.0, v1.1 | A Container Storage Interface (CSI) plugin for Zadara VPSA Storage Array & VPSA All-Flash  | Persistent | Read/Write Multiple Pods | Yes | Raw Block, Snapshot, Expansion, Cloning
+[Huawei Storage CSI](https://github.com/Huawei/eSDK_K8S_Plugin) | `csi.huawei.com` | v1.0 | A Container Storage Interface (CSI) Driver for [Huawei Storage](https://support.huawei.com/enterprise/en/category/enterprise-storage-pid-1482606575525?submodel=21430818, https://support.huawei.com/enterprise/en/category/cloud-storage-pid-1482615290338?submodel=21430817) | Persistent | Read/Write Multiple Pod | Yes |
```

Comment:
Can you put this in alphabetical order in the list?

Reply:
Also on the github page, can you provide instructions on how to install the csi driver?

Reply:
1. The new added content is in alphabetical now.
2. The document "User Guide.pdf" has been added to https://github.com/Huawei/eSDK_K8S_Plugin/tree/master/doc.